client/state/notices/README.md

```diff
@@ -0,0 +1,53 @@
+Notices
+===========
+
+A module for managing global notices.
+
+## Create a notice
+
+Provides some helper functions to render a notice on the UI. Types of notices are `error`, `success`.
+
+### How to use?
+
+#### Connect to Redux
+First, the component that wants to send notices must be connected to Redux.
+
+```javascript
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { successNotice, errorNotice } from 'state/notices/actions';
+
+...
+
+export default connect(
+	null,
+	dispatch => bindActionCreators( { successNotice, errorNotice }, dispatch )
+)( Component );
+```
+
+#### Uses
+
+Now you can use notices like so:
+
+```javascript
+
+this.props.successNotice(
+    'Settings saved successfully!', {
+    	duration: 4000
+} );
+
+```
+
+
+#### The available methods are:
+
+* `notices.success()`
+* `notices.error()`
```

Comment:
Yes, now I see these copy-paste errors :)
Attention to detail is not my forte :/
Will fix them!